mobile/Features/CheckIn/Mood.jsx

```diff
@@ -0,0 +1,191 @@
+import { useState } from 'react';
+import {
+  Text, View, Pressable, Image,
+} from 'react-native';
+import { useRoute } from '@react-navigation/native';
+import ProgressBar from 'react-native-progress/Bar';
+import PropTypes from 'prop-types';
+import nicole from '../../assets/sleepFace.png';
+import styles from './MoodStyle';
+
+function Mood({ navigation }) {
+  // get numPages from route, set progress to 1 / numpages
+  const route = useRoute();
+  const numPages = route.params?.numPages;
+  const progress = 1 / numPages;
+  // addedMoods array to keep track of the new moods the user has added
+  const [addedMoods, setAddedMoods] = useState([]);
+
+  const continueButton = () => {
+    // console.log('continue');
+    navigation.navigate('Sleep', { numPages });
+  };
+
+  const skipButton = () => {
+    // console.log('skip');
+  };
+
+  // later implement functionality for pressing on a mood button:
+
+  // const pressMood = () => {
+  //   console.log('mood');
+  // };
+
```

Comment:
while this is ok; could we have a state that keeps track of which mood was selected? like 47 vs no vs helppppp

Reply:
and have the pressMood console.log the mood that is in that state (so uncomment + modify)